backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/service/SAPGLJournalCopyRequest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * #%L
+ * de.metas.acct.base
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.acct.gljournal_sap.service;
+
+import de.metas.acct.gljournal_sap.SAPGLJournalId;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+import java.time.Instant;
+
+@Value
+@Builder
+public class SAPGLJournalCopyRequest
+{
+	@NonNull SAPGLJournalId sourceJournalId;
+
+	@NonNull Instant dateDoc;
+
+	@NonNull Boolean negateAmounts;
```

Comment:
why not primitive boolean?

Reply:
used Boolean and NonNull to be sure that the client is obliged to set it and is aware of its value (i.e. we don't end up with default mistaken values)